server/routes/establishmentRollRouter.ts

```diff
@@ -19,6 +19,7 @@ export default function establishmentRollRouter(services: Services): Router {
   )
 
   get('/', establishmentRollController.getEstablishmentRoll())
+  get('/locations/', establishmentRollController.getEstablishmentRoll(true))
```

Comment:
This is the path to a secret screen to show the establishment roll for new locations data